code/datums/supply_packs/black_market.dm

```diff
@@ -0,0 +1,1308 @@
+		/obj/item/clothing/head/cmcap/boonie
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+//Misc
+
+/datum/supply_packs/contraband/deep_storage/pdtl_kit
+	name = "PDT/L Kit"
+	contains = list(
+		/obj/item/storage/box/pdt_kit
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/lucky_lime
+	name = "Frozen Lime"
+	contains = list(
+		/obj/item/reagent_container/food/snacks/grown/lime
+		)
+	dollar_cost = 5
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/uno_reverse_random
+	name = "Uno Reverse Card"
+	randomised_num_contained = 1
+	contains = list(
+		/obj/item/toy/handcard/uno_reverse_red,
+		/obj/item/toy/handcard/uno_reverse_blue,
+		/obj/item/toy/handcard/uno_reverse_purple,
+		/obj/item/toy/handcard/uno_reverse_yellow
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/jungle_boots
+	name = "Jungle Pattern Combat Boots"
+	contains = list(
+		/obj/item/clothing/shoes/marine/jungle
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+//Weapons
+
+/datum/supply_packs/contraband/deep_storage/type_replica
+	name = "Type 80 Bayonet Replica"
+	contains = list(
+		/obj/item/attachable/bayonet/upp_replica
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/cartridge_bayonet
+	name = "M8 Cartridge Bayonet Kit"
+	contains = list(
+		/obj/item/storage/box/c02_knife
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/m43_holdout
+	name = "M43 Holdout Pistol"
+	contains = list(
+		/obj/item/storage/box/m43
+		)
+	dollar_cost = 10
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/m4a3_c
+	name = "M4A3 Custom Pistol"
+	contains = list(
+		/obj/item/weapon/gun/pistol/m4a3/custom
+		)
+	dollar_cost = 35
+	containertype = /obj/structure/largecrate/black_market
+
+/datum/supply_packs/contraband/deep_storage/m44_c
+	name = "M44 Custom Revolver"
+	contains = list(
+		/obj/item/weapon/gun/revolver/m44/custom
+		)
+	dollar_cost = 70
+	containertype = /obj/structure/largecrate/black_market
+
+/* --- MISCELLANEOUS --- */
+
+/*
+
+Things that don't fit anywhere else. If they're meant for shipside use, they probably fit in shipside contraband.
+
+*/
+
+/datum/supply_packs/contraband/miscellaneous
+	group = "Miscellaneous"
+
+/datum/supply_packs/contraband/miscellaneous/secured_wildlife
+	name = "secured wildlife container"
+	dollar_cost = 45
+	containertype = /obj/structure/largecrate/black_market/secured_wildlife
+
+/obj/structure/largecrate/black_market/secured_wildlife
+	name = "secured wildlife container"
+	icon_state = "lisacrate"
+	var/mob/living/contained_mob
```

Comment:
why does this need to be a var on the crate

If it stays, QDEL_NULL it on destroy and code doc it